process-services/latest/config/index.md

```diff
@@ -45,55 +45,41 @@ By default, the following properties are defined:
+> **Note:** The generation of `ENCRYPTEDPASSWORD` should be done on the JVM of Process Services.
 
-2. Extract the zip file and grant yourself full run permissions on its `bin` directory.
-
-    You need to know what encryption algorithms are supported. If you’re using the JVM to which the application will be deployed you can do this using the listAlgorithms tool that Jasypt provides: [http://www.jasypt.org/cli.html](http://www.jasypt.org/cli.html){:target="_blank"}.
-
-    >**Note:** Certain algorithms such as `SHA1 (PBEWITHSHA1ANDDESEDE)` and `MD5 (PBEWithMD5AndDES)` are available on most JVMs but more secure algorithms require modifications to the JRE policies.
-
-3. Choose an algorithm.
+1. Choose an algorithm.
 
     If you do not specify an algorithm to Jasypt, then you effectively obtain the default of `PBEWithMD5AndDES`. Some algorithms may appear in the list but may not be usable as the JRE policy blocks them.
 
-    If you want to increase your range of choices then you can modify the JRE policies: [https://www.ca.com/us/services-support/ca-support/ca-support-online/knowledge-base-articles.tec1698523.html](https://www.ca.com/us/services-support/ca-support/ca-support-online/knowledge-base-articles.tec1698523.html){:target="_blank"}. There is an equivalent for the IBM JRE: [https://www-01.ibm.com/marketing/iwm/iwm/web/reg/pick.do?source=jcesdk.](https://www-01.ibm.com/marketing/iwm/iwm/web/reg/pick.do?source=jcesdk){:target="_blank"}.
+    If you want to increase your range of choices then you can modify the JRE policies: [Default Policy Implementation and Policy File Syntax](https://docs.oracle.com/javase/8/docs/technotes/guides/security/PolicyFiles.html){:target="_blank"}. There is an equivalent for the IBM JRE: [IBM WebSphere Java unrestricted policy files](https://www.ibm.com/support/pages/ibm-websphere-java-unrestricted-policy-files){:target="_blank"}.
 
     Algorithms using AES are generally considered most secure. TripleDES also passes security checks at present. You should consult your security department for advice specific to your organization and the needs of your server.
 
-4. Use the Jasypt tools to encrypt the value.
-
-    You can use the encrypt script that comes with Jasypt to encrypt the value against your chosen secret password. In addition to their documentation, see this guide: [http://www.programering.com/a/MjN1kTNwATg.html](http://www.programering.com/a/MjN1kTNwATg.html){:target="_blank"}.
-
-    We recommend to avoid using quotes. Also check that you can decrypt the value, preferably using the intended JRE.
-
-5. Deploy the application.
-
-    See the application installation instructions.
+2. Use the JVM of APS.
 
-6. Set the value in the properties file.
+    Move into your working directory `cd /usr/local/tomcat/webapps/activiti-app/WEB-INF/lib` and then run `java -cp jasypt-1.9.3.jar org.jasypt.intf.cli.JasyptPBEStringEncryptionCLI input=<database password> password=secretpassword algorithm=PBEWithMD5AndDES`.
```

Comment:
Think this is too specific (`cd /usr/local/tomcat/webapps/activiti-app/WEB-INF/lib`).

I'm repeating myself but adding this is more generic:
```
If, for example, you are using Tomcat on Unix, you can go to `tomcat_home/webapps/activiti-app/WEB-INF/lib` and then run `java -cp jasypt-1.9.3.jar org.jasypt.intf.cli.JasyptPBEStringEncryptionCLI input=<database password> password=secretpassword algorithm=PBEWithMD5AndDES`.

```

Reply:
Changed